### DISCUSSION TITLE
Indent Blankline doesn't render for Go file

### DISCUSSION THREAD

**Discussion by takadev15** [lukas-reineke/indent-blankline.nvim#905] [Q&A] [answered]
I've been using indent blankline for a while and lately i've been having issue with it on a Golang file. Usually in other file ex. rust the indent blankline was rendered like shown in the image below

![image](https://github.com/lukas-reineke/indent-blankline.nvim/assets/59208568/5d4a148a-706e-4a3a-95df-0cdc39020f6e)

But for Go file for some reasone the line isn't rendered, and this only happens after using a formatter like go fmt, gofumpt, or cloning a fresh go file form remote repository. 

![image](https://github.com/lukas-reineke/indent-blankline.nvim/assets/59208568/2c283696-e9b3-4423-81a8-6db860fde23d)

N.B. The shiftwidth and tabstop is set to 2, that's why the indentation looks close with each other here. 

I presume this because Go (and hence the formatter) use tab for indentation and each tab is recognized as one spaces and that's what made this isn't recognized as one tab indent by the plugins, because if i indent it manually by clicking tab (which recognized as 2 or 4 spaces depending on the tabstop config) the indentation rendered successfully. 

I just want to make sure if any solutions or maybe someone here had similar problem as this

Thanks 😄

**Comment by lukas-reineke** [comment]
Please use the [minimal reproduction template](https://github.com/lukas-reineke/indent-blankline.nvim/wiki/Minimal-reproduction-template) to reproduce this. It is very likely a configuration issue somewhere in your config.

**Comment by takadev15** [comment]
![image](https://github.com/lukas-reineke/indent-blankline.nvim/assets/59208568/8f67d42c-3241-4d8c-bbe2-007801ba06eb)

Yep looks like you're right. Using minimal config does render the indent line. So probably the bug was caused by configuration issue in the indent-blankline config that i have. 

Do you have any suggestion on where to look out in the config regarding this?

**Reply by lukas-reineke** [reply, marked as answer]
How the tab char is decided is documented under `:help ibl.config.indent.tab_char`

**Reply by takadev15** [reply]
Yep you're right. Looks like my config (which i don't set the tab_chars manually) doesn't set the chars for tab automatically. I set the `ibl.config.indent.tab_char` manually then the indent was rendered successfully. This make me wonders on the manpage that was said that the default tab_chars should be set the same as chars, looks like this not happens on my config tough

**Reply by lukas-reineke** [reply]
> This make me wonders on the manpage that was said that the default tab_chars should be set the same as chars, looks like this not happens on my config tough

Only if you don't have `'list'` set.

**Reply by takadev15** [reply]
ah i see, looks like that's what broke my config. Well then thanks lukas for the help
